server/zanata-model/src/main/java/org/zanata/hibernate/search/AbstractFieldBridge.java

```diff
@@ -0,0 +1,68 @@
+package org.zanata.hibernate.search;
+
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.FieldType;
+import org.apache.lucene.index.IndexOptions;
+import org.hibernate.search.bridge.FieldBridge;
+import org.hibernate.search.bridge.LuceneOptions;
+
+/**
+ * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
+ */
+public abstract class AbstractFieldBridge implements FieldBridge {
```

Comment:
Don't think I will make anymore changes in this PR. But I will make sure the subsequent PR will be per warning.